addons/website_event_sale/controllers/main.py

```diff
@@ -22,14 +22,26 @@ def _create_attendees_from_registration_post(self, event, registration_data):
         # we have at least one registration linked to a ticket -> sale mode activate
         if not any(info.get('event_ticket_id') for info in registration_data):
             return super()._create_attendees_from_registration_post(event, registration_data)
+        # using a set comprehension at the start so that it doesn't browse 2 times the same record
+        # also adding the "if data.get('event_ticket_id')" condition so that we don't consider the ones without tickets
```

Comment:
Hi @adsh-odoo :smile: 
Thanks for updating!
You can remove those 2 lines :slightly_smiling_face:  I wrote them to make sure you understood what I meant :smile: 
Cheers!

Reply:
Hello @amdi-odoo
Oh misunderstood that one :sweat_smile: 
Removed 
Thanks!!